src/Dashboard/Dashboard.php

```diff
@@ -478,10 +478,10 @@ public static function checkRights(array $rights = []): bool
 
        // check specific rights
         if (
-            count(array_intersect($rights['entities_id'], $_SESSION['glpiactiveentities']))
-            || in_array($_SESSION["glpiactiveprofile"]['id'], $rights['profiles_id'])
-            || in_array($_SESSION['glpiID'], $rights['users_id'])
-            || count(array_intersect($rights['groups_id'], $_SESSION['glpigroups']))
+            count(array_intersect($rights['entities_id'], $_SESSION['glpiactiveentities'] ?? []))
```

Comment:
i suggest to add before the set of checks a `Session::getLoginUserID()` instead of all coalescing null

Reply:
In the case of a shared dashboard to be read anonymously, `Session::getLoginUserID()` will always return false?

Reply:
In case there is no user logged-in, none of the checked session variables should be set, so none of these checks should return `true`.